1_fetch/src/fetch_dv_data.R

```diff
@@ -0,0 +1,33 @@
+#' @title Download the discharge from NWIS for each dv gage
+#' 
+#' @param ind_file character file name where the output should be saved
+#' @param sites_ind indicator file for the vector of site numbers
+#' @param dates object from viz_config.yml that specifies dates as string
+fetch_dv_data <- function(ind_file, sites_ind, dates){
+  
+  sites <- readRDS(scipiper::sc_retrieve(sites_ind, remake_file = '1_fetch.yml'))
+  
+  dv_sites_data <- lapply(sites, FUN = function(x){
```

Comment:
I guess this works, but couldn't this just be done with fewer web service calls and then some dplyr?

Reply:
👍 I'll need to think about it a bit more, but there was some reason we wanted things in this format before. Maybe it's less relevant now that we aren't using D3 + Canvas. I will add an issue to rethink, but optimization of this part really isn't the game for this particular development effort since we know this works.

Reply:
sounds good.